cmd/generate/generate.go

```diff
@@ -0,0 +1,59 @@
+/*
+Copyright 2020-2021 The OpenEBS Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package generate
+
+import (
+	"strings"
+
+	"github.com/openebs/openebsctl/pkg/generate"
+	"github.com/openebs/openebsctl/pkg/util"
+	"github.com/spf13/cobra"
+)
+
+// NewCmdGenerate provides options for generating
+func NewCmdGenerate() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:       "generate",
+		ValidArgs: []string{"cspc"},
+		Short:     "Generate one or more OpenEBS resources based on flags",
+	}
+	cmd.AddCommand(NewCmdGenerateStorage())
+	return cmd
+}
+
+func NewCmdGenerateStorage() *cobra.Command {
```

Comment:
nit: Good to have a function name more specific to CSPC/cStor pool instead of a generic name(in the future we can accommodate mayastor pools as well)
```suggestion
func NewCmdGenerateCstorStoragePool() *cobra.Command {
```

Reply:
I'll make this change as we aren't sure if mayastor functionalities will be included in this CLI.

Reply:
Changed it to `NewCmdGenerateCstorStoragePoolCluster`